AUTHORS

```diff
@@ -1,70 +1,197 @@
 Fernando Matos <fernando@pixele.com.br>
 fsbcorrea <fsbcorrea@gmail.com>
-Gustavo <gugahoa@hotmail.com>
+Gabriel <gabriel.gmartins091@gmail.com>
+Gabriel Pedro <gpedro@users.noreply.github.com>
+Gubihe <korejwo.michal@gmail.com>
+gunzino <gunzo@gunzo.eu>
+Gustavo Aguiar <gugahoa@gmail.com>
 Gustavo Stor <gsa2@cin.ufpe.br>
+idontreallywolf <nedim.kanat1999@gmail.com>
+Iñaki Amatria Barral <34030065+amatria@users.noreply.github.com>
+infernumx <spaghetti90314@gmail.com>
+jerryskye <jerryskye@users.noreply.github.com>
+Jerzy Skalski <payments.otsme@gmail.com>
 Jesus Fernandez <jfevia@gmail.com>
+Joseluis González <7506164+joseluis2g@users.noreply.github.com>
+Joseph Bingham <jo3bingham@users.noreply.github.com>
+Josue David Hernandez <megadavidtor@gmail.com>
 Jovial1991 <jovial1991@hotmail.com>
-jpmagg <jamison.usa@hotmail.com>
-Kamil Chojnowski <diath@zodiaclabs.org>
+jpmagg <jpmatheus.agg@hotmail.com>
+Kairion <github@kairion.eu.org>
+Kamenuvol <luizvitor@hotmail.com.br>
 Kamil Wąż <sn3ejk@outlook.com>
+Kevin Zou <17015060+nkzou@users.noreply.github.com>
+Klaffen <kent_96@live.no>
 Kornelijus Survila <kornholijo@gmail.com>
+KrecikOnDexin <krecikondexin@gmail.com>
+kygo <kenovski6@gmail.com>
+Leo32 <bwils120@hotmail.com>
+Luan Cuba <luancuba9@gmail.com>
+Luan Luciano <luanluciano@outlook.com>
+luca <681992+lukka@users.noreply.github.com>
+Lucas CP <lucascercondesrad@gmail.com>
+lucasgrizante <40500081+lucasgrizante@users.noreply.github.com>
 Luckey1729 <admin@gmghosting.net>
 LuisPro <ramayahero@wp.pl>
-Marcin <nekiroboy@gmail.com>
+maattch <51377408+maattch@users.noreply.github.com>
+mackerel225 <davidb9966@gmail.com>
+Malizia R <rmalizia44@gmail.com>
 Marcin Michalski <evulmastah@gmail.com>
+Márcio Porto <44486916+marcinho1994@users.noreply.github.com>
+Marco Oliveira <marcomoa@hotmail.com>
 margohq <muno376@gmail.com>
+Mario Tettei <Shadow280690@hotmail.de>
 Mark Samman <mark.samman@gmail.com>
 Markus Elfring <elfring@users.sourceforge.net>
+Martin Nylind <3932953+forgee@users.noreply.github.com>
+Mateuso8 <47433262+Mateuso8@users.noreply.github.com>
 mattyx14 <martin_jojo@hotmail.com>
+Mazen Mardini <mazen@mengate.se>
+Michal Kulesza <michal.kulesza@netguru.co>
+Mikołaj Herwart <m.herwart@gmail.com>
+MillhioreBT <sarahelizabetwesker@gmail.com>
 Mithryll <guianm@outlook.com>
+Mkalo <mkalo@xtibia.com>
+Moisés dos Santos <40324910+lyuz1n@users.noreply.github.com>
+mrianura123 <46069341+mrianura123@users.noreply.github.com>
 Nailson <nailsonnego@gmail.com>
 nawyrus <nawyrus@users.noreply.github.com>
 nclx <januario.nclx@gmail.com>
+Nekiro <nekiro.dev@gmail.com>
 octavio007 <octavioargenis_11@hotmail.com>
+Patryk Zajdler <pzajdler2@gmail.com>
 paulosalvatore <paulo.salvatore@hotmail.com.br>
+Pegason <49088829+pegason@users.noreply.github.com>
 PrinterLUA <roberto_potrus@hotmail.com>
-Ranieri Althoff <ranisalt@gmail.com>
+Ramon Bernardo <ramonbernardo.contato@gmail.com>
+Ranieri Althoff <ranisalt@protonmail.com>
```

Comment:
```suggestion
Ranieri Althoff <1993083+ranisalt@users.noreply.github.com>
```

How did you get that email? :thinking:

Reply:
> How did you get that email? 🤔

only GitHub knows 😄 
`git log --all --format='%aN <%aE>' | sort -u | uniq > AUTHORS`